docs/content.md

```diff
@@ -0,0 +1,43 @@
+# MITRE ATT&CK Threat Hunting
+
+# Overview
+
+The MITRE ATT&CK Threat Hunting Solution Pack demonstrates a variety of scenarios and use cases around threat hunting using the information provided by the MITRE ATT&CK Framework. 
+
+*MITRE ATT&CK is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations. The ATT&CK knowledge base issued as a foundation for the development of specific threat models and methodologies in the private sector, in government, and in the cybersecurity product and service community.*
```

Comment:
why new file? can we add this in same readme?

Reply:
Deleted